app/embeds/GoogleDriveVideo.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import * as React from "react";
+import Frame from "./components/Frame";
+
+const URL_REGEX = new RegExp(
+  "^https?://drive.google.com/file/d/(.*)/view$"
+);
+
+type Props = {|
+  attrs: {|
+    href: string,
+    matches: string[],
+  |},
+|};
+
+export default class GoogleDriveVideo extends React.Component<Props> {
```

Comment:
Let's make it `GoogleDrive`, there's nothing video specific around these links as lots of file types can be contained on Drive